packages-content-model/roosterjs-content-model-dom/lib/formatHandlers/common/borderFormatHandler.ts

```diff
@@ -42,15 +49,27 @@ export const borderFormatHandler: FormatHandler<BorderFormat> = {
 
         if (borderRadius) {
             format.borderRadius = borderRadius;
+        } else {
+            BorderRadiusKeys.forEach(key => {
+                const value = element.style[key];
+
+                if (value) {
+                    format[key] = value;
+                }
+            });
         }
     },
     apply: (format, element) => {
-        BorderKeys.forEach(key => {
+        BorderKeys.concat(BorderRadiusKeys).forEach(key => {
```

Comment:
Let make the result of "BorderKeys.concat(BorderRadiusKeys)" to be a global constant so no need to calculate them every time.

Reply:
Sure, done.